src/test/java/com/ibm/cloud/cloudant/kafka/SourceChangesConnectorTest.java

```diff
@@ -48,7 +53,18 @@ protected void setUp() throws Exception {
     /**
      * Test method for {@link SourceChangesConnector#stop()}.
      */
+    @PrepareForTest(CachedClientManager.class)
+
     public void testStop() {
+        Map<String, Cloudant> clientCache = new ConcurrentHashMap<>();
+        clientCache.put(sourceProperties.get("name"), PowerMock.createMock(Cloudant.class));
+        Whitebox.setInternalState(CachedClientManager.class, "clientCache", clientCache);
```

Comment:
You could also use the approach of other tests which use `ClientManagerUtils.addClientToCache` to access the cache which is package-private. You'd also need to write a helper for your cache empty asserts.

Reply:
Great idea! I have fixed this in fc2798d